### PR TITLE
[Hotfix]: Sync Implementation With Design

### PR DESCRIPTION
**Pull request by mojtabaesl**
# What?

- Alert component has been added
- Input minHeight has been removed from theme.ts
- Notification styles have been modified
- Unnecessary styles have been removed

# Why?
Our implementations of UI weren't the same as UI

### CODE REVIEW

File: cypress/e2e/createProject.cy.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ describe('Create Project', () => {
     cy.findByTestId(createProjectFormIds.notification.success).should('exist');
   });
 
-  it('[NOK]: Admin wants to create a project with short name', () => {
+  it.skip('[NOK]: Admin wants to create a project with short name', () => {
```
Comment (AmirabbasJ) [inline cypress/e2e/createProject.cy.ts:28]: why is this skipped? 🤔

File: src/components/Alert/Alert.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  component: Alert,
+} as ComponentMeta<typeof Alert>;
+
+const Template: ComponentStory<typeof Alert> = args => <Alert {...args} />;
+
+export const Error = Template.bind({});
+Error.args = {
+  message: errorMessages.UNKNOWN_ERROR,
+  type: 'error',
+};
```
Comment (AmirabbasJ) [inline src/components/Alert/Alert.stories.tsx:16]: what about other states?

Comment (mojtabaesl) [reply src/components/Alert/Alert.stories.tsx:16]: we don't have them for now.

File: cypress/e2e/createProject.cy.ts (hunk 1 above)
Comment (mojtabaesl) [reply cypress/e2e/createProject.cy.ts:28]: I haven't understood this test
and we mark it as `NOK`

Comment (AmirabbasJ) [reply cypress/e2e/createProject.cy.ts:28]: the NOK means that the test is for a situation in the app that should not work for 
in other words, the admin wants to create a project with a short name but that's not ok (NOK)

Comment (mojtabaesl) [reply cypress/e2e/createProject.cy.ts:28]: but the test is ok, right?
I mean we're typing 3 letters word

File: src/components/CreateProject/ProjectForm/CreateProjectForm.tsx [hunk 3]
```diff
@@ -72,10 +56,23 @@ export const CreateProjectForm = ({ dismiss }: Props) => {
       });
 
       if (isNull(data)) throw new Error('data is null');
-      notifySuccessCreation(data.createProject.name);
+      // notifySuccessCreation(data.createProject.name);
```
Comment (ASafaeirad) [inline src/components/CreateProject/ProjectForm/CreateProjectForm.tsx:59]: Remove this line please

**Review by ASafaeirad — APPROVED** [review]

File: cypress/e2e/createProject.cy.ts (hunk 1 above)
Comment (AmirabbasJ) [reply cypress/e2e/createProject.cy.ts:28]: yes if I understood correclty 
we're typing 3 letters which should result in a error (NOK)

**Review by AmirabbasJ — APPROVED** [review]

File: cypress/e2e/createProject.cy.ts (hunk 1 above)
Comment (mojtabaesl) [reply cypress/e2e/createProject.cy.ts:28]: but 3 letters word is ok
I think Alireza just changed the title

**Review by erfan-goodarzi — APPROVED** [review]

**Pull request merged**